src/app/containers/CpsOnwardJourney/index.jsx

```diff
@@ -76,6 +76,14 @@ const LegacyGridItemConstrainedLarge = styled(GridItemConstrainedLarge)`
 
 const StyledSectionLabel = styled(SectionLabel)`
   margin-top: 0;
+  ${props =>
+    props.labelId === 'top-stories-heading' &&
```

Comment:
Could we look into why we need to vary the css for this specific `labelId`? This is a generic component so does not really want to be [coupled](https://en.wikipedia.org/wiki/Loose_coupling) to a specific piece of content. Feel free to jump on a call if you want to talk through it.